File: Python_Ruby/conditional/4.rb
input_password = 11

real_egoing_password = 11
real_k8805_password = "ab"

if real_egoing_password == input_password
    puts("Hello!, egoing")
else
    if real_k8805_password == input_password
        puts("Hello!, k8805")
    else
        puts("Who are you")
    end
end

File: Python_Ruby/container_loop/4.py
input_password = input("Please input your password: ")

# real_egoing_password = "egoing"
# real_k8805_password = "k8805"
members = ['egoing', 'k8805']

# for member in members:
#     if member != input_password:
#         if member == members[len(members) - 1]:
#             print("Who are you")
#         continue
#     else:
#         print("Hello!, " + member)
#         break

for member in members:
    if member == input_password:
        print('Hello!, ' + member)
        import sys
        sys.exit()
print('Who are you?')